examples/supplychain-app/charts/expressapp-quorum/values.yaml

```diff
@@ -16,7 +16,15 @@ deployment:
   #   annotations:
   #     key: "value"
   annotations: {}
-
+vault:
```

Comment:
Add comments and example for each key here.

Reply:
ok will add that